code/game/objects/items/scanners.dm

```diff
@@ -72,7 +72,7 @@ REAGENT SCANNER
 	origin_tech = "magnets=1;biotech=1"
 	var/mode = 1
 	var/hud_mode = 1
-	var/skill_threshold = SKILL_MEDICAL_PRACTICED
+	var/skill_treshhold = SKILL_MEDICAL_PRACTICED
```

Comment:
Why changing `threshold` to `treshhold`?

Reply:
As you can see by the commits it wasn't intentional

Reply:
Ah, typo.